sceptre/connection_manager.py

```diff
@@ -92,22 +92,23 @@ class ConnectionManager(object):
     _clients = {}
     _stack_keys = {}
 
-    def __init__(self, region, profile=None, stack_name=None, iam_role=None):
+    def __init__(self, region, profile=None, stack_name=None, iam_role=None, iam_role_session_duration=3600):
```

Comment:
Let's make the 3600 a properly named constant on this module, I think. Something like DEFAULT_IAM_SESSION_DURATION, or something like that.